pom.xml

```diff
@@ -800,8 +524,8 @@
                 </configuration>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-resources-plugin</artifactId>
-                <version>2.6</version>
```

Comment:
version is 2.7 in parent pom. Not necessarily a problem but might be worth checking?